data/hai/unfettered.txt

```diff
@@ -0,0 +1,271 @@
+
+phrase "hostile unfettered challenger"
+	word
+		"Now is not the time for talk"
+		"Yes! Fight me"
+		"Our battle will be sang in legends for years to come"
+		"Do not try to distract me with hails"
+		"Do you want to surrender already? Ha"
+	word
+		"!"
+
+phrase "hostile unfettered challenger"
+	word
+		"Do you wish to end the fight?"
+		"I can see you are having difficulties. Feel free to call this off for another time..."
+		"Remember, you only need to give me a symbolic bribe for this to stop."
+		"This is not a duel to the death, or else only one of us would be able to learn."
+		"Do you wish to surrender?"
+		"I am reasonable. If you wish to surrender, we can come to terms."
+		"You are lucky that I have to accept any ceasefire if you ask... you would do well not to attempt anything."
+
+phrase "hostile unfettered challenger"
+	word
+		"Here's a "
+	word
+		"free"
+		"useful"
+		"common"
+	word
+		" tip: "
+		" hint: "
+	word
+		"when feeling overwhelmed in duels you can always ${overwhelmed in duel}! Try it next time."
+		"challengers have common weaknesses, and you should try to get an edge by ${advantage in duel}."
+		
+
+phrase "overwhelmed in duel"
+	word
+		"tactically retreat to the edge of the system"
+		"bribe the challengers"
+		"try to split the enemies apart"
+		"go to another system"
+		"upgrade your ship design"
+		"scan the enemy to find out their vulnerabilities"
+
+phrase "advantage in duel"
+	word
+		"figuring out if their engines are weak"
+		"trying to slide away whilst shooting them back"
+		"using your range advantage, be it in missiles, ion cannons or else"
+		"try to get your hands on powerful prototypes"
+		"go and acquire alien technology. Even human ones have their uses"
+		"overheat their ship"
+		"shoot them down with missiles if they lack sufficient jammers and anti-missiles"
+		"try to get at their batteries with ion weapons"
+
+phrase "hostile unfettered challenger"
+	word
+		"If you happen to know of "
+		"If you heard of "
+		"When you come to know about "
+		"Next time you get your hands on "
+		"If you 'find' "
+	word
+		"a strong"
+		"an interesting"
+		"a good"
+		"an exotic"
+		"a respectable"
+		"a powerful"
+		"an overpowered"
+	word
+		" weapon combo"
+		" alien weapon"
+		" human junk-weapon"
+	word
+		", you should come and "
+	word
+		"demonstrate it to us in action"
+		"fight me with it"
+		"sell it to us, after a proper demonstration"
+	word
+		"."
+	word
+		" You might find some of us willing to buy it, for the proper price"
+	word
+		"!"
+
+phrase "defeated unfettered challenger"
+	word
+		"You have "
+		"You managed to have "
+		"Somehow, you have "
+		"I must admit you have "
+	word
+		"beaten "
+		"defeated "
+		"ashamed "
+		"disabled "
+		"revealed my weakness% "
+	word
+		"me in a% "
+	word
+		"glorious "
+		"pathetic "
+		"exceptional "
+		"grandiose "
+		"fast-paced "
+		"strategic "
+	word
+		"battle"
+		"combat"
+		"duel"
+		"fight"
+		"challenge"
+	word
+		"."
+		"!"
+	word
```

Comment:
Would it be simpler to put the space after the !/.? and remove the space from before everything else?